easystacks/software.eessi.io/2023.06/zen4/eessi-2023.06-eb-4.9.3-2023b.yml

```diff
@@ -0,0 +1,2 @@
+easyconfigs:
+  - Valgrind-3.23.0-gompi-2023b.eb
```

Comment:
The bot didn't actually build this version, it seems it was already built and installed for zen4 (in https://github.com/EESSI/software-layer/pull/614). Can you remove this easystack file form the PR?